Makefile

```diff
@@ -343,6 +353,10 @@ unit-test: KUBEBUILDER_ASSETS ?= $(shell $(SETUP_ENVTEST) use --use-env -p path
 unit-test:
 	KUBEBUILDER_ASSETS="$(KUBEBUILDER_ASSETS)" $(GO_TEST) ./... -cover -tags "$(BUILD_TAGS)"
 
+.PHONY: local-e2e
+local-e2e: e2e ## Run e2e test's locally
```

Comment:
curious what this is for
it seems like it does the same thing as `capd-test`

Reply:
Yeah it pretty much is but a more generic name with generic variable to override whereas capd test is very specific. I can update capd to call the generic target?

Reply:
I feel like we can just remove `capd-test`